playbooks/cluster_setup_ha.yaml

```diff
@@ -79,27 +95,196 @@
     tmpdir: "/tmp"
   tasks:
     - block:
-        - name: Install corosync configuration 
+        - name: Install corosync configuration
           copy:
             src: "{{ tmpdir }}/corosync.conf"
             dest: /etc/corosync/corosync.conf
             owner: root
             group: root
             mode: '0644'
-        - name: Install corosync key 
+        - name: Install corosync key
           copy:
             src: "{{ tmpdir }}/authkey"
             dest: /etc/corosync/authkey
             owner: root
             group: root
             mode: '0400'
-        - name: Start pacemaker
+        - name: Start corosync
           ansible.builtin.systemd:
               name: corosync
               state: started
               enabled: true
       when: groups['valid_machine'] is defined
 
+# Configure qDevice and qNetd
```

Comment:
Could this be put in a sub-playbook or in roles ? This is a lot of tasks, and the file would be >300 lines long

Reply:
Also it would allow having one check on the `corosync_use_qdevice` variable